### DISCUSSION TITLE
Error when using admin view

### DISCUSSION THREAD

**Discussion by alan123489** [joomengine/Joomla-Component-Builder#943] [Q&A] [answered]
When I am using component builder everything works fine until I try to edit the fields from the admin view.

When I

Admin view
Fields
Edit (to add more fields)

I get

An error has occurred.

This occurs on all admin views.  I'm using firefox 106.0.4 (64-bit), php 7.4 and Joomla and component builder are both up to date.

0 Argument 1 passed to VDM\Joomla\Utilities\GetHelper::between() must be of the type string, null given, called

Any help would be great

**Comment by Llewellynvdm** [comment, marked as answer]
For JCB related bugs, and other JCB core issues please use: https://git.vdm.dev/joomla/Component-Builder/issues
